libs/design-system/src/lib/ipRegex.ts

```diff
@@ -1,7 +1,11 @@
-// Simpler combined regex https://regex101.com/r/0WMysi/2
+const domain = '((?!-)[A-Za-z0-9-]+([-.]{1}[a-z0-9]+)*.[A-Za-z]{2,6})'
```

Comment:
Is it worth the hassle to validate this client side? Calling the endpoint will try to connect. If it's not valid, it will fail anyway.

Reply:
Its a little extra immediate feedback but I agree we could just get rid of it.

What about updating the blocklist, should I remove there too? Does the server-side validate the addresses being added via `/bus/hosts/blocklist`? cc @ChrisSchinnerl

Reply:
Its extra immediate feedback which I think is nice for the user, but I agree at least in this simple connect form we could get rid of it as the error will be pretty clear.

What about updating the blocklist, should I remove there too? Does the server-side validate the addresses being added via `/bus/hosts/blocklist`? cc @ChrisSchinnerl 

I feel like might as well leave them both, rather than removing, unless you foresee this causing any issues.

Reply:
No, I don't think the blocklist does. It just compares whatever string you provide to whatever string the host announced itself as. e.g. if a host announces as "some.invalid:string" you are able to block that string and prevent that host from being scanned (even though it would fail anyway) even though the address you blocked isn't a valid one.

Reply:
@ChrisSchinnerl that is good to know, cc @n8maninger - so would you both rather I remove validation in these two places or leave it? Let me know and I will update the PR.

Reply:
I think removing the validation is fine in both cases